docs.json

```diff
@@ -69,8 +69,17 @@
                     "group": "Available Connections",
                     "pages": [
                       "agents/available-connections",
-                      "agents/connections/supabase",
-                      "agents/connections/neo4j"
+                      {
+                        "group": "CRMs",
```

Comment:
Let's make sure these sub-groups match the categories used on the "Available Connections" page. So the sub-group names here would be "Sales" and "Database"